pipeline/runtime/common.go

```diff
@@ -24,15 +24,26 @@ func getNudges() []logstream.Nudge {
 	}
 }
 
-func getOutputVarCmd(outputVars []string, outputFile string) string {
+func getOutputVarCmd(entrypoint, outputVars []string, outputFile string) string {
 	cmd := ""
 	for _, o := range outputVars {
-		cmd += fmt.Sprintf(";echo \"%s $%s\" >> %s", o, o, outputFile)
+		env := fmt.Sprintf("$%s", o)
+		if isPowershell(entrypoint) {
```

Comment:
What if we don't have any entrypoint (running directly on VM)?

Reply:
This method won't be called. Output variable does not make sense if entrypoint is not present. It is a plugin step instead.

Reply:
it errs out here: https://github.com/harness/lite-engine/pull/35/files#diff-b07c32047a7ce106899517b16b9e24c271e15b45b78e514c4dd34b5d790e570fL24